### DISCUSSION TITLE
Terrestria Rubber Wood

### DISCUSSION THREAD

**Discussion by ghost** [TechReborn/TechReborn#2517] [Ideas]
**Is your feature request related to a problem? Please describe.**
[Terrestria](https://www.curseforge.com/minecraft/mc-mods/terrestria) also adds rubber trees, but I can't use them like TR rubber logs. This clearly isn't a bug, but interoperability between these two popular Fabric mods would be great.

**Describe the solution you'd like**
I would like to be able to gather sap from Terrestria rubber trees, or at least to extract rubber from them in centrifuges.

**Comment by modmuss50** [comment]
Extracting rubber from them should be easy to add.

**Comment by DanShu15** [comment]
Basalt from Terrestria to dust would be neat too. If it already works I couldn't figure out the recipe or how to do it

**Comment by Sturmlilie** [comment]
@DanShu15 It's actually already supported in Tech Reborn, but still blocked on the Terrestria side of things: https://github.com/TerraformersMC/Terrestria/pull/117

**Comment by saltyseadoggo** [comment]
Linking the issue on Terrestria's page: https://github.com/TerraformersMC/Terrestria/issues/205#issue-743331628
